invenio_rdm_records/config.py

```diff
@@ -176,3 +176,10 @@ def _(x):
 RDM_RECORDS_DOI_DATACITE_PASSWORD = ""
 RDM_RECORDS_DOI_DATACITE_PREFIX = "10.1234"
 RDM_RECORDS_DOI_DATACITE_TEST_MODE = True
+
+
+# Invenio-IIIF
+# ===================
+
+IIIF_API_DECORATOR_HANDLER = None
```

Comment:
These won't be picked up (Invenio-IIIF might initialize before RDM-Records and ext.py is not loading them). Can you remove from this PR and move to Invenio-App-RDM config.py instead?